Lab 2/Lab2.ipynb

```diff
@@ -0,0 +1,861 @@
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.625"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "stats.norm.ppf(1-prob)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6ee1eb74-7cf0-4806-a886-70db16218d11",
+   "metadata": {},
+   "source": [
+    "0.625$\\sigma$ is a pitifully low value! This is far below our convention of 5$\\sigma$ being a significant discovery, and it is much more likely that the measurement came from the background."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b8550b7-40e7-453a-8104-2f1d22f14e4e",
+   "metadata": {},
+   "source": [
+    "## Version 2.\n",
+    "\n",
+    "### B.\n",
+    "If you have to look for a signal in 10k pixels, what is the probability distribution of your background?\n",
+    "\n",
+    "To understand how we tackle this question, state the statistical question at hand\n",
+    "\n",
+    "### Question: What is the probability of finding a signal of our candidate, over a sample of 10k trials (in this case, 10k pixels)?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "68cc884e-cf97-473e-9525-7305de920c6e",
+   "metadata": {},
+   "source": [
+    "We can reevaluate our initial probability distribution by multiplying the overall curve by ten thousand."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 204,
+   "id": "562638b2-0500-44af-b100-64b4cffe6b5b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<matplotlib.lines.Line2D at 0x7fe746373d00>]"
+      ]
+     },
+     "execution_count": 204,
+     "metadata": {},
+     "output_type": "execute_result"
+    },
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAd8AAAFQCAYAAAD+5nRVAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjQuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/Z1A+gAAAACXBIWXMAAAsTAAALEwEAmpwYAAAs4UlEQVR4nO3deXScd33v8fdX+2It3iTL8prYhDgLJJgQIL0sZYkDbegtLUlvGkppc0MToJdSGmhv9wJdTuFCaXIC5LZQTnM4bVpcCFCWApclEGcljrPIjhdZu23tGkmj+d4/ZkaeyGNrRp55npl5Pq9zdKx5lplvnhzpo9/v+f1+j7k7IiIiEpyqsAsQERGJGoWviIhIwBS+IiIiAVP4ioiIBEzhKyIiEjCFr4iISMAUviKSMzP7qpm9PcdjD5vZ64pdk0g5UviKVCAzu93M9pnZrJn9wzmO+5CZTaa+Yma2kPF6/9Lj3X2Pu/9jUYsXiQCFr0hl6gP+HLjnXAe5+4fdfZW7rwJuBX6Ufu3ul6SPsyT9vhApEP0wiVQgd7/P3f8dOLHS9zCz75jZX5jZD4Bp4ILUtt9I7b/QzL5tZifMbMTMvmBm7QX5DxCpcApfETmXXwVuAVqAI0v2GfARYCNwMbAZ+OMgixMpVwpfETmXf3D3/e4ed/f5zB3u3uPu33D3WXcfBv4WeFU4ZYqUl5qwCxCRknbsbDvMrAP4BPAzJFvGVcCpgOoSKWtq+YrIuZzrsWcfSe2/3N1bgZtIdkWLyDIUviIVyMxqzKwBqAaqzazBzArd09UCTAKjZtYN/G6B31+kYil8RSrTHwAzwB0kW6QzqW2F9CfAlcAY8BXgvgK/v0jFMvdz9SqJiIhIoanlKyIiEjCFr4iISMAUviIiIgFT+IqIiARM4SsiIhKwil3hat26db5t27awyxARkYh66KGHRtx9fbZ9FRu+27ZtY9++fWGXISIiEWVmSx9GskjdziIiIgFT+IqIiARM4SsiIhIwha+IiEjAFL4iIiIBU/iKiIgELLDwNbNrzexpM+sxszuy7H+hmf3IzGbN7P35nCsiIlJOAglfM6sGPgXsAXYBN5rZriWHnQTeA/zNCs4VEREpG0G1fK8Cetz9kLvPAfcC12ce4O5D7v4gMJ/vuSIiIuUkqPDtBo5lvO5NbSv2uSKSp9n4Al97op9Hjp4KuxS
```

Comment:
I think you've done this section wrong. The problem wants you to consider how many times you would expect to see your measurement in 10k pixels. So you don't want to normalize your 10k PDF. Instead, consider how your original measurement had a 26% chance of being measured in 1 pixel. Over 10k pixels, you would expect to get this value .26 * 10,000 times -- and you can therefore say that this is DEFINITELY not a significant measurement. If your measurement instead originally had 5 sigma likelihood, after you multiply that probability by 10,000, the probability will still be less than 1, but it will no longer be a 5 sigma measurement.